docs/release-notes/release-notes-0.20.0.md

```diff
@@ -28,6 +28,10 @@
 
 ## lncli Additions
 
+* [`lncli sendpayment --keysend` and `lncli queryroutes` now support the
```

Comment:
needs to be changed because we will allow also `amp` payments not only keysend ...